.gitignore

```diff
@@ -137,3 +138,9 @@ src/sonic-daemon-base/sonic_daemon_base.egg-info
 # Misc. files
 files/initramfs-tools/arista-convertfs
 files/initramfs-tools/union-mount
+
+# object files
+src/**/*.o
+
+# debian working dir
+src/dhcpmon/debian
```

Comment:
You are actually committing files in this directory in this PR. If you want to ignore everything else that gets generated in that d-irectory, you should probably explicitly un-ignore those files (refer to the systemd-sonic-generator example on lines 85-90 above). Also, this should be moved into that list above and inserted in alphabetical order.

Reply:
Reworked the inclusion and exclusion. .gitignore will no prevent commit files in ignored dir, one has to use -f to do so.

Reply:
>.gitignore will no prevent commit files in ignored dir, one has to use -f to do so

I'm not sure what you mean by this. You can ignore the directory then subsequently un-ignore specific files within that directory. There are multiple examples in this file.